Client/src/pages/sign_in.vue

```diff
@@ -1,68 +1,83 @@
 <template>
-  <q-page class="column items-center">
+  <q-page class="self-center">
```

Comment:
I think it looks better with "items-center" instead of "self-center"

Reply:
this is done